code/modules/mob/living/carbon/xenomorph/evolution.dm

```diff
@@ -303,10 +303,15 @@
 		if(new_xeno)
 			qdel(new_xeno)
 		return
-	if(tier != XENO_TIER_ONE || !regression )
-		new_xeno.upgrade_xeno(upgrade, TRUE)
-	if(!regression && upgrade != XENO_UPGRADE_INVALID)
-		new_xeno.upgrade_xeno(new_xeno.upgrade_prev(), TRUE)
+
+	if(!regression)
+		if(upgrade != XENO_UPGRADE_INVALID)
+			new_xeno.upgrade_xeno(upgrade_prev(), TRUE)
+	else
+		if(tier == XENO_TIER_ONE)
+			new_xeno.upgrade_xeno(XENO_UPGRADE_INVALID, TRUE)
+		else
+			new_xeno.upgrade_xeno(upgrade_next(), TRUE)
```

Comment:
this was removed intentionally, because that creates exploit

Reply:
> this was removed intentionally, because that creates exploit

What kind of exploit?

Reply:
I tested it in many ways. And btw it's commit on RUTGMC

Reply:
You can get primo iirc. There was another one with queen devo, cant remember.

Also this meant that going from one t3 to another had no cost, which wasnt really balanced imo